GramAddict/plugins/interact_hashtag_posts.py

```diff
@@ -0,0 +1,264 @@
+import logging
+from functools import partial
+from random import seed, shuffle
+
+from colorama import Fore, Style
+from GramAddict.core.decorators import run_safely
+from GramAddict.core.device_facade import DeviceFacade
+from GramAddict.core.filter import Filter
+from GramAddict.core.interaction import (
+    _on_interaction,
+    _on_like,
+    _on_watch,
+    interact_with_user,
+    is_follow_limit_reached_for_source,
+)
+from GramAddict.core.plugin_loader import Plugin
+from GramAddict.core.scroll_end_detector import ScrollEndDetector
+from GramAddict.core.storage import FollowingStatus
+from GramAddict.core.utils import get_value, random_sleep
+from GramAddict.core.views import (
+    TabBarView,
+    HashTagView,
+    OpenedPostView,
+    PostsViewList,
+    Swipe_to,
+)
+
+logger = logging.getLogger(__name__)
+
+# Script Initialization
+seed()
+
+
+class InteractHashtagLikers(Plugin):
+    """Handles the functionality of interacting with a hashtags post owners"""
+
+    def __init__(self):
+        super().__init__()
+        self.description = (
+            "Handles the functionality of interacting with a hashtags post owners"
+        )
+        self.arguments = [
+            {
+                "arg": "--hashtag-posts",
+                "nargs": "+",
+                "help": "interact to hashtag post owners in recent tab",
+                "metavar": ("hashtag1", "hashtag2"),
+                "default": None,
+                "operation": True,
+            },
+            {
+                "arg": "--interact-chance",
```

Comment:
I'm fighting between changing the name and making this a core argument. Is there other places an interact-chance would make sense?

Reply:
everything that will be done in feed..
for example the plugin for interact with your own feed

Reply:
Never mind 😆

Reply:
lol souds right to me :D